core/metamodel/src/main/java/org/apache/isis/core/metamodel/facets/members/cssclassfa/annotprop/CssClassFaFacetOnMemberFactory.java

```diff
@@ -76,25 +76,27 @@ public void process(ProcessContributeeMemberContext processMemberContext) {
 
     private static CssClassFaFacet createFromMetadataPropertiesIfPossible(
             final ProcessContextWithMetadataProperties<? extends FacetHolder> pcwmp) {
-        
+
         final FacetHolder holder = pcwmp.getFacetHolder();
-        
+
         final Properties properties = pcwmp.metadataProperties("cssClassFa");
         return properties != null ? new CssClassFaFacetOnMemberFromProperties(properties, holder) : null;
     }
 
-    private CssClassFaFacet createFromAnnotationIfPossible(final ProcessMethodContext processMethodContext) {
-        final CssClassFa annotation = Annotations.getAnnotation(processMethodContext.getMethod(), CssClassFa.class);
-        return annotation != null ? new CssClassFaFacetOnMemberAnnotation(annotation.value(), annotation.position(), processMethodContext.getFacetHolder()) : null;
-    }
+    // private CssClassFaFacet createFromAnnotationIfPossible(final ProcessMethodContext processMethodContext) {
+    // final CssClassFa annotation = Annotations.getAnnotation(processMethodContext.getMethod(), CssClassFa.class);
+    // return annotation != null ? new CssClassFaFacetOnMemberAnnotation(annotation.value(), annotation.position(),
+    // processMethodContext.getFacetHolder()) : null;
+    // }
```

Comment:
Why this is commented out?
If the reason is good then it should be removed, not just commented out.